### ISSUE TITLE
Question of loss of performance

### ISSUE THREAD

**Issue by sshhj89** [bach05/SOODImageNet#2]
Thank you for your good datasets.

I am interested in using your datasets but I have a question about the loss of performance.

"The third column reveals that the models perform worse on C^H_ood compared to C^E_ood, which is semantically closer to the training set,
with at least a 24% drop in F1 score."

It doesn't match with the result in Table 1. For example, Renset50 has 0.62 and 0.46 on each out-of-distribution dataset respectively. The loss of performance should be like 16.xx. 

Could you clarify it?

**Comment by bach05** [comment]
Hello,

In Table 1, we reported the percentage variation of the F1 score in C<sup>H</sup><sub>ood</sub> compared to C<sup>E</sup><sub>ood</sub>.
In other words, the variation is computed as:

```math
\Delta = \frac{  F1(C^{H}_{ood}) - F1(C^{E}_{ood})}{F1(C^{E}_{ood})}
```

We chose this approach to highlight the relative difference in performance between the hard and easy out-of-distribution (OOD) sets. 

I hope this clarify! If you have further questions, please do not hesitate to contact us.
